docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+
+2. [JdbcPollInputOperator](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPollInputOperator.java): It uses String for the generic parameter. This operator defines a single port and processes each database table record one by one as String tuple. Details are discussed below.
+
+## Partitioning of JDBC Poller
+#### Static Partitioning
+Only static partitioning is supported for JDBC Poller Input Operator. Configure parameter `partitionCount` to define the desired number of initial partitions (4 in this example).
+**Note**: The additional partition will be created to poll newly added records.
+
+```xml
+  <property>
+    <name>dt.operator.{OperatorName}.prop.partitionCount</name>
+    <value>4</value>
+  </property>
+```
+
+where {OperatorName} is the name of the JDBC Poller operator.
+
+This will create 5 operator instances in all. Four of these will fetch the data which is currently in the table. We call these static non-polling partitions. An additional partition will be created which will read any newly added data. We call such a partition as a polling partition, as it "polls" for newly added data. Changing the value of the property will create corresponding number of static partitions.
+
+#### Dynamic Partitioning
+Not supported.
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***3.5.0***
+3. Operator state: ***Evolving***
+4. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcPollInputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.html)***
+
+JDBC Poller is **idempotent**, **fault-tolerant** and **statically partitionable**.
+
+## AbstractJdbcPollInputOperator
+This is the abstract implementation that serves as base class for polling messages from jdbc store. It can be extended to modify functionality or add new capabilities. This class doesn’t have any ports, so concrete subclasses will need to provide them if necessary.
+
+![AbstractJdbcPollInputOperator.png](images/jdbcinput/operatorsClassDiagram.png)
+
+###<a name="AbstractJdbcPollInputOperatorProps"></a>Properties of AbstractJdbcPollInputOperator
+Several properties are available to configure the behavior of this operator and they are summarized in the table below. Of those only store, tableName, columnsExpression and key are mandatory ones. Those properties can be set like this:
```

Comment:
I would prefer not to include such details. There is already a discussion to change nature of query to use lower and greater than rather than limit and offset kind of queries. Such changes in implementation needs to be in consistent with document. I would prefer to avoid such data in document which goes in depth of coding or design.